File: 2025-1/sesion-de-clase-18/SoftProgNet/SoftProgPersistencia/DAO/RRHH/IEmpleadoDAO.cs
using System.Collections.Generic;
using PUCP.Edu.Pe.SoftProg.Modelo.RRHH;

namespace PUCP.Edu.Pe.SoftProg.Persistencia.DAO.RRHH {
    public interface IEmpleadoDAO : ICrud<Empleado> {
        List<Empleado> BuscarPorDni(string dni);
    }
}

File: 2025-1/sesion-de-clase-20/SoftProg/SoftProgPersistencia/src/main/java/pe/edu/pucp/inf30/softprog/dao/rrhh/IEmpleadoDAO.java
package pe.edu.pucp.inf30.softprog.dao.rrhh;

import pe.edu.pucp.inf30.softprog.dao.ICrud;
import pe.edu.pucp.inf30.softprog.model.rrhh.Empleado;

/**
 *
 * @author eric
 */
public interface IEmpleadoDAO extends ICrud<Empleado> {
    Empleado buscarPorDni(String dni);
    // No hay metodos extra por el momento
    // Metodos especificos de Empleado seran agregados aqui
}
